discovery_engine/lib/src/domain/feed_manager.dart

```diff
@@ -74,6 +80,11 @@ class FeedManager {
             });
 
           final feed = sortedActives.map((doc) => doc.toApiDocument()).toList();
+
+          if (feed.isEmpty) {
```

Comment:
When the user start the app the feed is empty and is not error. Also this is a breaking change.

Reply:
The user can also decide to close all the documents in the feed.

Reply:
I reverted this back to previous state.

If the empty case is not meant to be represented by `FeedRequestFailed` event, then maybe we don't need it at all?